internal/run/convert.go

```diff
@@ -44,6 +65,18 @@ func NewConvert(cfg env.Config, kubeConfig string, kubeContext string) *Convert
 		cfg.MaxReleaseVersions = 10
 	}
 
+	if cfg.TillerNS == "" {
+		cfg.TillerNS = "kube-system"
+	}
+
+	if cfg.TillerLabel == "" {
+		cfg.TillerLabel = "OWNER=TILLER"
+	}
+
+	if cfg.Release != "" {
```

Comment:
The upgrade `Prepare` method [handles error checking for required inputs](https://github.com/mongodb-forks/drone-helm3/blob/master/internal/run/upgrade.go#L58-L63). This should probably do something similar, rather than just accept an empty release name.

Reply:
moved the check to Prepare, thx